api/src/middleware/__tests__/programsRouter.ts

```diff
@@ -211,4 +211,48 @@ describe('programsRouter', () => {
         .expect(500, done);
     });
   });
+
+  describe('GET /activityStatus/:programId', () => {
+    it('should return a list of activities and their completed statuses', done => {
```

Comment:
It should return the program ID and a list of the program's activities and the principal's completed statuses.